autogenerate_api_client.sh

```diff
@@ -0,0 +1,18 @@
+curl -X POST https://generator3.swagger.io/api/generate -H 'content-type: application/json' -d '{ "specURL" : "https://tdei-gateway-dev.azurewebsites.net/v3/api-docs", "lang" : "typescript-axios", "type" : "CLIENT", "codegenVersion" : "V3"}' --output tdei-client-axios.zip
+unzip -qo tdei-client-axios.zip
+
+if which git >/dev/null 2>&1; then
+  git branch -D scratch/api-client
```

Comment:
why `scratch/api-client` branch?

Reply:
@santaduda @susrisha 
Firstly, I used it as an example locally to verify the script was working.
Secondly, I mentioned in the call to either directly commit to the `main` branch or let someone review and merge it as PR although I prefer to do the latter as everything is automated. This is a choice we have to make as a team.

Finally, good point on bumping the npm version. We should do that!